code/game/atoms.dm

```diff
@@ -132,8 +132,8 @@
 /atom/proc/is_open_container()
 	return flags & OPENCONTAINER
 
-atom/proc/is_semi_open_container()

```

Comment:
typo

also are you sure TRANSPARENT isn't defined elsewhere? could cause issues